db/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 
 [dependencies]
 color-eyre = { workspace = true }
+reqwest = "0.12.4"
```

Comment:
reqwest crate for downloading the RAW HTML

Reply:
Ahh! So I think this is why `cargo deny` is blowing up for you, its doing its job for use here lol 🎉 😆 

One of the big reasons I use `cargo deny` is to prevent pulling in `OpenSSL`. Which is a C library that has been used basically everywhere in the last to do `ssl` which powers `https`

But its a C library which can make compiling the Rust harder, and there is a Rust only alternative that I like to use! And even better `reqwest` can work with either SSL package! So we just need to tell `reqwest` that we want to use different features

https://doc.rust-lang.org/cargo/reference/features.html

Here is where I do reqwest in another project. I think we can leave off the `json` here since we aren't parsing json here!
https://github.com/coreyja/coreyja.com/blob/main/Cargo.toml#L28C1-L31C30

Basically I think we want the `default-features-false` and then the `features` list with `rustls-tls`!

Reply:
Interesting stuff! It's crazy how we can tell Rust which one to use and not to use